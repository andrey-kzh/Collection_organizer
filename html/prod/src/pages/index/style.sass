@import ../../css/mixins.sass
@import ../../css/variables.sass
