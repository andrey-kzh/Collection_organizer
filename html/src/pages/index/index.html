<!DOCTYPE html>
<html lang="ru">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Collection</title>
    <script src="/js/min/main.min.js"></script>
    <link href="/css/style.css" rel="stylesheet">
  </head>
  <body>
    <div class="content">
      <!--menu-->
      <div class="nav">
        <!--navLinkSearch-->
        <!--include ../../components/navLinkSearch/index-->
        <button class="nav-button-search" onclick="showHideDiv('selcat')">Поиск</button>
        <!--navLinkSearch end-->
        <div class="nav-wrap">
          <!--navLink--><a class="nav-link" href="#">Добавить</a><a class="nav-link" href="#">Настройка</a><a class="nav-link" href="#">Выход</a>
          <!--navLink end-->
        </div>
      </div>
      <!--menu end-->
      <!--search form-->
      <div class="search-form-wrap">
        <form class="search-form">
          <div class="search-form__input-wrap">
            <input class="search-form__input" type="text" name="title" placeholder="Название">
            <button class="search-form__button" type="submit">Найти</button>
            <!--categorys-->
            <div class="categorys">
              <!--addCategoryButton-->
              <button class="categorys-add-button">Категории</button>
              <!--addCategoryButton end-->
              <!--categoryItem-->
              <div class="categorys-items-wrap">
                <label class="category-item">
                  <div class="category-item__title">Компьютеры</div>
                  <input class="category-item__input" type="checkbox" checked name="categitem[]" value="123">
                  <button class="category-item__close"></button>
                </label>
                <label class="category-item">
                  <div class="category-item__title">Диски</div>
                  <input class="category-item__input" type="checkbox" checked name="categitem[]" value="123">
                  <button class="category-item__close"></button>
                </label>
                <label class="category-item">
                  <div class="category-item__title">1990е</div>
                  <input class="category-item__input" type="checkbox" checked name="categitem[]" value="123">
                  <button class="category-item__close"></button>
                </label>
                <label class="category-item">
                  <div class="category-item__title">Комплектующие</div>
                  <input class="category-item__input" type="checkbox" checked name="categitem[]" value="123">
                  <button class="category-item__close"></button>
                </label>
              </div>
              <!--categoryItem end-->
            </div>
            <!--categorys and-->
          </div>
        </form>
      </div>
      <!--search form end-->
      <!--catalog lent-->
      <div class="catalog">
        <div class="catalog-item">
          <div class="catalog-item__img" style="background-image:url(&quot;/img/486pc.jpg&quot;);"></div>
          <div class="catalog-item__txt">
            <div class="catalog-item__header">Материнская плата socket 370</div>
            <div class="catalog-item__categorys">
              <div class="catalog-item__category-item">1990е</div>
              <div class="catalog-item__category-item">Компьютеры</div>
              <div class="catalog-item__category-item">Игры</div>
              <div class="catalog-item__category-item">Комплектующие</div>
            </div>
            <div class="catalog-item__anons">Вот вам яркий пример современных тенденций - укрепление и развитие внутренней структуры однозначно фиксирует необходимость дальнейших направлений развития. Не следует, однако, забывать, что базовый вектор развития предоставляет широкие возможности для благоприятных перспектив.</div>
          </div>
          <div class="catalog-item__ctrl-buttons">
            <!--button-->
            <button class="button button_catalog-item">Редактировать</button>
            <button class="button button_catalog-item">Удалить</button>
            <!--button end-->
          </div>
        </div>
        <div class="catalog-item">
          <div class="catalog-item__img" style="background-image:url(&quot;/img/486pc.jpg&quot;);"></div>
          <div class="catalog-item__txt">
            <div class="catalog-item__header">Dungeon Keeper</div>
            <div class="catalog-item__categorys">
              <div class="catalog-item__category-item">1990е</div>
              <div class="catalog-item__category-item">Компьютеры</div>
              <div class="catalog-item__category-item">Игры</div>
              <div class="catalog-item__category-item">Комплектующие</div>
            </div>
            <div class="catalog-item__anons">Вот вам яркий пример современных тенденций - укрепление и развитие внутренней структуры однозначно фиксирует необходимость дальнейших направлений развития. Не следует, однако, забывать, что базовый вектор развития предоставляет широкие возможности для благоприятных перспектив.</div>
          </div>
          <div class="catalog-item__ctrl-buttons">
            <!--button-->
            <button class="button button_catalog-item">Редактировать</button>
            <button class="button button_catalog-item">Удалить</button>
            <!--button end-->
          </div>
        </div>
        <div class="catalog-item">
          <div class="catalog-item__img" style="background-image:url(&quot;/img/486pc.jpg&quot;);"></div>
          <div class="catalog-item__txt">
            <div class="catalog-item__header">Sound Blaster Pro</div>
            <div class="catalog-item__categorys">
              <div class="catalog-item__category-item">1990е</div>
              <div class="catalog-item__category-item">Компьютеры</div>
              <div class="catalog-item__category-item">Игры</div>
              <div class="catalog-item__category-item">Комплектующие</div>
            </div>
            <div class="catalog-item__anons">Вот вам яркий пример современных тенденций - укрепление и развитие внутренней структуры однозначно фиксирует необходимость дальнейших направлений развития. Не следует, однако, забывать, что базовый вектор развития предоставляет широкие возможности для благоприятных перспектив.</div>
          </div>
          <div class="catalog-item__ctrl-buttons">
            <!--button-->
            <button class="button button_catalog-item">Редактировать</button>
            <button class="button button_catalog-item">Удалить</button>
            <!--button end-->
          </div>
        </div>
        <div class="catalog-item">
          <div class="catalog-item__img" style="background-image:url(&quot;/img/486pc.jpg&quot;);"></div>
          <div class="catalog-item__txt">
            <div class="catalog-item__header">486 компьютер</div>
            <div class="catalog-item__categorys">
              <div class="catalog-item__category-item">1990е</div>
              <div class="catalog-item__category-item">Компьютеры</div>
              <div class="catalog-item__category-item">Игры</div>
              <div class="catalog-item__category-item">Комплектующие</div>
            </div>
            <div class="catalog-item__anons">Вот вам яркий пример современных тенденций - укрепление и развитие внутренней структуры однозначно фиксирует необходимость дальнейших направлений развития. Не следует, однако, забывать, что базовый вектор развития предоставляет широкие возможности для благоприятных перспектив.</div>
          </div>
          <div class="catalog-item__ctrl-buttons">
            <!--button-->
            <button class="button button_catalog-item">Редактировать</button>
            <button class="button button_catalog-item">Удалить</button>
            <!--button end-->
          </div>
        </div>
      </div>
      <!--catalog lent end-->
      <!--selectCategorys-->
      <div class="popup-wrap" id="selcat" style="display:none;">
        <div class="popup">
          <div class="popup__close-wrap">
            <button class="popup__close" onclick="showHideDiv('selcat')"></button>
          </div>
          <div class="popup__content">
            <div class="select-category-header">Категории</div>
            <div class="select-category select-category_none">
              <!--selectCategoryItem-->
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Диски</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">1990е</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Комплектующие</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Комплектующие</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Компьютеры</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Диски</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">1990е</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Тест</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Игры</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Аудиокассеты</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Видеокассеты</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Компьютеры</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Диски</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">1990е</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Комплектующие</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Комплектующие</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Компьютеры</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Диски</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">1990е</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Тест</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Игры</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Аудиокассеты</span>
              </label>
              <label class="select-category-item">
                <input class="select-category-item__input" type="checkbox" name="categitem[]" value="123"><span class="select-category-item__title">Видеокассеты</span>
              </label>
              <!--selectCategoryItem end-->
            </div>
          </div>
        </div>
      </div>
      <!--selectCategorys end-->
    </div>
  </body>
</html>