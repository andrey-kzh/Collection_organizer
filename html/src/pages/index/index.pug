doctype html
html(lang="ru")
    head
        meta(charset="utf-8")
        title="Collection"
        block scripts
            script(src='/js/min/main.min.js')
        link(href='/css/main.css', rel='stylesheet')

    body
        div(class="content")

            //menu
            include ../../components/nav/index
            //menu end
