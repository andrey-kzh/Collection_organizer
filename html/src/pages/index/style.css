.content {
  max-width: 850px;
  margin: 0 auto;
  padding: 58px 0 0 0; }

.nav {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
      -ms-flex-pack: justify;
          justify-content: space-between;
  -webkit-box-align: baseline;
  -webkit-align-items: baseline;
      -ms-flex-align: baseline;
          align-items: baseline;
  max-width: 650px; }

.nav-wrap {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
      -ms-flex-pack: end;
          justify-content: flex-end; }

a.nav-link {
  display: block;
  padding: 6px 8px;
  text-decoration: none;
  color: #000000;
  margin: 0 0 0 1px;
  border: 1px #ffffff solid;
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  -webkit-transition: all 0.1s linear 0s;
  -o-transition: all 0.1s linear 0s;
  transition: all 0.1s linear 0s;
  font-size: 1rem; }

a.nav-link:hover,
a.nav-link_active {
  color: #BDBDBD;
  border: 1px #BDBDBD solid;
  border-radius: 13px; }

a.nav-link-search {
  display: block;
  position: relative;
  padding: 6px 16px 6px 26px;
  text-decoration: none;
  color: #000000;
  margin: 0;
  border: 1px #ffffff solid;
  border-radius: 13px;
  -webkit-transition: all 0.1s linear 0s;
  -o-transition: all 0.1s linear 0s;
  transition: all 0.1s linear 0s;
  font-size: 1rem; }

a.nav-link-search:hover {
  color: #BDBDBD;
  border: 1px #BDBDBD solid;
  border-radius: 13px; }

a.nav-link-search::before {
  content: '';
  display: block;
  border: 6px solid transparent;
  border-left: 12px solid #000000;
  -webkit-transform-origin: 6px 6px;
      -ms-transform-origin: 6px 6px;
          transform-origin: 6px 6px;
  -webkit-transition: all 0.1s linear 0s;
  -o-transition: all 0.1s linear 0s;
  transition: all 0.1s linear 0s;
  position: absolute;
  left: 10px;
  top: 8px;
  -webkit-box-sizing: border-box;
          box-sizing: border-box; }

a.nav-link-search:hover:before {
  border-left: 12px solid #BDBDBD; }

.nav-link-search_rotate90:before {
  -webkit-transform: rotate(90deg);
      -ms-transform: rotate(90deg);
          transform: rotate(90deg); }

.nav-link-search_rotate0:before {
  -webkit-transform: rotate(0deg);
      -ms-transform: rotate(0deg);
          transform: rotate(0deg); }

.nav-button-search {
  display: block;
  position: relative;
  padding: 6px 16px 6px 26px;
  text-decoration: none;
  color: #000000;
  margin: 0;
  border: 1px #ffffff solid;
  border-radius: 13px;
  -webkit-transition: all 0.1s linear 0s;
  -o-transition: all 0.1s linear 0s;
  transition: all 0.1s linear 0s;
  background-color: transparent;
  font-size: 1rem;
  cursor: pointer; }

.nav-button-search:hover {
  color: #BDBDBD;
  border: 1px #BDBDBD solid;
  border-radius: 13px; }

.nav-button-search::before {
  content: '';
  display: block;
  border: 6px solid transparent;
  border-left: 12px solid #000000;
  -webkit-transform-origin: 6px 6px;
      -ms-transform-origin: 6px 6px;
          transform-origin: 6px 6px;
  -webkit-transition: all 0.1s linear 0s;
  -o-transition: all 0.1s linear 0s;
  transition: all 0.1s linear 0s;
  position: absolute;
  left: 10px;
  top: 8px;
  -webkit-box-sizing: border-box;
          box-sizing: border-box; }

.nav-button-search:hover:before {
  border-left: 12px solid #BDBDBD; }

.nav-link-search_rotate90:before {
  -webkit-transform: rotate(90deg);
      -ms-transform: rotate(90deg);
          transform: rotate(90deg); }

.nav-link-search_rotate0:before {
  -webkit-transform: rotate(0deg);
      -ms-transform: rotate(0deg);
          transform: rotate(0deg); }
