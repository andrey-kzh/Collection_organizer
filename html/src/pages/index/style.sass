@import ../../css/mixins.sass
@import ../../css/variables.sass

.content
  max-width: 1000px
  margin: 0 auto
  padding: 58px 20px 0 20px
  @media screen and (max-width: $mobileResolution)
    padding: 10px 20px 0 20px

//Import components style
@import "../../components/nav/style.sass"
@import "../../components/navLink/style.sass"
@import "../../components/navLinkSearch/style.sass"
@import "../../components/navButtonSearch/style.sass"
@import "../../components/searchForm/style.sass"
@import "../../components/categorysAddButton/style.sass"
@import "../../components/categorys/style.sass"
@import "../../components/categorys/style.sass"
@import "../../components/categorysItem/style.sass"
@import "../../components/catalogLent/style.sass"
@import "../../components/button/style.sass"
@import "../../components/popup/style.sass"
@import "../../components/selectCategorys/style.sass"
@import "../../components/selectCategorysItem/style.sass"
