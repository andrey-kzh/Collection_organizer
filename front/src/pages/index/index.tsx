import * as React from 'react';
import './style.sass';

import {Nav} from '../../components/nav'
import {SearchForm} from '../../components/searchForm'

export const IndexPage: React.FC = () => {

    return <>
        <Nav/>
        <SearchForm/>
          </>;
}