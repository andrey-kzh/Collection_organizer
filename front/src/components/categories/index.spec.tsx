import React from 'react'
import { mount, shallow, render } from 'enzyme';
import { Categories } from './index';

describe('Categories', () => {

    const props = {
        selectdedCategories: [12, 13],
        delSelectdedCategories: () => { },
        addSelectdedCategories: () => { }
    }
    const categoriesFullState = {
        items: {
            12: { id: 12, title: 'testCategoriesTitle_01' },
            13: { id: 13, title: 'testCategoriesTitle_02' },
            14: { id: 14, title: 'testCategoriesTitle_03' }
        },
        list: [12, 13, 14]
    }
    const mockGetAllCategories = jest.fn()
    const context = (categoriesState: {} | boolean) => {
        return {
            categoriesStore: {
                categories: categoriesState,
                getAllCategories: mockGetAllCategories
            }
        }
    }

    afterEach(() => {
        jest.clearAllMocks()
    })

    it('Should render html snapshot', () => {
        jest.spyOn(React, 'useContext').mockReturnValue(context(categoriesFullState))
        const component = shallow(<Categories {...props} />)
        expect(component).toMatchSnapshot()
    })

    it('Should request categories data and render loading stab', () => {
        jest.spyOn(React, 'useContext').mockReturnValueOnce(context(null))
        const component = mount(<Categories {...props} />)
        expect(mockGetAllCategories).toHaveBeenCalledTimes(1)
        expect(component.find('div').at(0).text()).toEqual('Loading')
    })

    it('Should render SelectCategories component when button is clicked', () => {
        jest.spyOn(React, 'useContext').mockReturnValue(context(categoriesFullState))
        const component = mount(<Categories {...props} />)
        component.find('.categorys-add-button').simulate('click')
        expect(component.find('.select-category').length).toEqual(1)
    })




})