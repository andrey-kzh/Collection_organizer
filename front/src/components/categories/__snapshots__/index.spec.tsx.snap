// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories Should render SelectCategories component when button is clicked 1`] = `
<Memo(observerComponent)
  addSelectdedCategories={[Function]}
  delSelectdedCategories={[Function]}
  selectdedCategories={
    Array [
      12,
      13,
    ]
  }
>
  <div
    className="categorys"
  >
    <CategoriesAddButton
      callback={[Function]}
    >
      <button
        className="categorys-add-button"
        onClick={[Function]}
      >
        Категории
      </button>
    </CategoriesAddButton>
    <div
      className="categorys-items-wrap"
    >
      <Component
        delCallback={[Function]}
        key="12"
        title="testCategoriesTitle_01"
      >
        <label
          className="category-item"
        >
          <div
            className="category-item__title"
          >
            testCategoriesTitle_01
          </div>
          <input
            className="category-item__input"
            name="category_item"
            type="checkbox"
          />
          <button
            className="category-item__close"
            onClick={[Function]}
          >
             
          </button>
        </label>
      </Component>
      <Component
        delCallback={[Function]}
        key="13"
        title="testCategoriesTitle_02"
      >
        <label
          className="category-item"
        >
          <div
            className="category-item__title"
          >
            testCategoriesTitle_02
          </div>
          <input
            className="category-item__input"
            name="category_item"
            type="checkbox"
          />
          <button
            className="category-item__close"
            onClick={[Function]}
          >
             
          </button>
        </label>
      </Component>
    </div>
  </div>
  <Memo(observerComponent)
    addSelectdedCategories={[Function]}
    delSelectdedCategories={[Function]}
    selectdedCategories={
      Array [
        12,
        13,
      ]
    }
    setIsOpen={[Function]}
  >
    <Popup
      className=""
      closeCallback={[Function]}
    >
      <div
        className="popup-wrap "
      >
        <div
          className="popup"
        >
          <div
            className="popup__close-wrap"
          >
            <button
              className="popup__close"
              onClick={[Function]}
            />
          </div>
          <div
            className="popup__content"
          >
            <div
              className="select-category-header"
            >
              Категории
            </div>
            <div
              className="select-category select-category_none"
            >
              <Component
                addCallback={[Function]}
                delCallback={[Function]}
                isActive={true}
                key="12"
                title="testCategoriesTitle_01"
              >
                <label
                  className="select-category-item"
                >
                  <input
                    checked={true}
                    className="select-category-item__input"
                    name="category_item"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="select-category-item__title"
                  >
                    testCategoriesTitle_01
                  </span>
                </label>
              </Component>
              <Component
                addCallback={[Function]}
                delCallback={[Function]}
                isActive={true}
                key="13"
                title="testCategoriesTitle_02"
              >
                <label
                  className="select-category-item"
                >
                  <input
                    checked={true}
                    className="select-category-item__input"
                    name="category_item"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="select-category-item__title"
                  >
                    testCategoriesTitle_02
                  </span>
                </label>
              </Component>
              <Component
                addCallback={[Function]}
                delCallback={[Function]}
                isActive={false}
                key="14"
                title="testCategoriesTitle_03"
              >
                <label
                  className="select-category-item"
                >
                  <input
                    checked={false}
                    className="select-category-item__input"
                    name="category_item"
                    onChange={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="select-category-item__title"
                  >
                    testCategoriesTitle_03
                  </span>
                </label>
              </Component>
            </div>
          </div>
        </div>
      </div>
    </Popup>
  </Memo(observerComponent)>
</Memo(observerComponent)>
`;

exports[`Categories Should render html snapshot 1`] = `
<Fragment>
  <div
    className="categorys"
  >
    <CategoriesAddButton
      callback={[Function]}
    />
    <div
      className="categorys-items-wrap"
    >
      <Memo()
        delCallback={[Function]}
        key="12"
        title="testCategoriesTitle_01"
      />
      <Memo()
        delCallback={[Function]}
        key="13"
        title="testCategoriesTitle_02"
      />
    </div>
  </div>
</Fragment>
`;
